R/stan_module.R

```diff
@@ -31,9 +32,9 @@ StanModule <- setClass(
 read_stan <- function(string) {
     system_file <- system.file("stanparts", string, package = "jmpost")
     if (file.exists(string)) {
-        out <- readLines(string)
+        out <- read_file(string)
     } else if (file.exists(system_file)) {
```

Comment:
@gowerc is it possible to replace the `file.exists(system_file)` with `is_file(system_file)`. `is_file` is a function from package `fs` and explicitly checks whether a file exists and not whether the provided path is TRUE. I created a relevant issue [here ](https://github.com/Genentech/jmpost/issues/39).